css/portal-dashboard/feedback/feedback-legend.less

```diff
@@ -0,0 +1,27 @@
+.feedbackBadgeLegend {
+  margin: 20px;
+
+  .feedbackBadgeLegend__heading {
+    font-size: 16px;
+    margin-bottom: 10px;
+  }
+
+  .feedbackBadgeLegend__list {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    overflow: visible;
+    width: 100%;
+
+    dd, dt {
+      flex: 0 0 6%;
+      font-weight: normal;
+      line-height: 1;
+      margin-bottom: 5px;
+      overflow: hidden;
+    }
+    dd {
+      flex: 0 0 94%;
```

Comment:
I always get worried when I see a layout percent like 94.  I assume this works in practice though?